modules/apps/journal/journal-web-test/src/testIntegration/java/com/liferay/journal/web/internal/display/test/JournalDisplayContextTest.java

```diff
@@ -59,6 +70,7 @@
 /**
  * @author Marco Galluzzi
  */
+@FeatureFlags("LPD-42452")
```

Comment:
@gergelyszalay, better to add the annotation only to the test method, instead of the whole class